assets/js/post_office_search.js

```diff
@@ -3,4 +3,9 @@ import { createRoot } from 'react-dom/client';
 import AddressSearch from '@18f/identity-address-search';
 
 const root = createRoot(document.getElementById('post-office-search'));
-root.render(React.createElement(AddressSearch));
+root.render(
+  React.createElement(AddressSearch, {
+    addressSearchURL: 'https://idp.dev.identitysandbox.gov/api/address_search',
+    locationsURL: 'https://idp.dev.identitysandbox.gov/api/usps_locations',
```

Comment:
Should we make these configurable through `config.yml` / `config.dev.yml` so someone could point to a locally running instance, or eventually point to production in the live site?

For example, maybe we could put `data-` attributes on [the `post-office-search` element](https://github.com/18F/identity-site/blob/ab41e7bf6c9be030afdeec676f8820941ccd386f/_includes/components/post-office-search.html#L2):

```html
<div
  id="post-office-search"
  data-address-search-url="{{ site.po_search_address_search_url }}"
  data-locations-search-url="{{ site.po_search_locations_search_url }}"
></div>
```

Then do something like what we do with [`AppRootData` in document capture](https://github.com/18F/identity-idp/blob/318be15269a1649ed48bf2ca7e5213b6c4a8de6b/app/javascript/packs/document-capture.tsx#L25) to specify the properties on the element's `dataset`.

Could even _replace_ the `show_po_search` option and only render the script if the URL configs are present, though I don't feel as strongly about that.

Reply:
+++ big fan of this approach, it's what we do for the international table support: https://github.com/18F/identity-site/blob/ab41e7bf6c9be030afdeec676f8820941ccd386f/_includes/country_support_table.html#L15

In fact, how stable are the paths? Could we reuse the same base URL config?